src/completers/git.zsh

```diff
@@ -303,6 +303,61 @@ _fzf_complete_git() {
         return
     fi
 
+    if [[ $subcommand = 'fetch' ]]; then
+        local prefix_option completing_option
+
+        local git_options_argument_required=(--depth --deepen -j --jobs --negotiation-tip -o --recurse-submodules-default --refmap --server-option --shallow-exclude --shallow-since --submodule-prefix --upload-pack)
+        local git_options_argument_optional=(--recurse-submodules -S)
+
+        if completing_option=$(_fzf_complete_parse_completing_option "$prefix" "$last_argument" ${(F)git_options_argument_required} ${(F)git_options_argument_optional}); then
```

Comment:
The quotations of `${(F)git_options_argument_required}` and `${(F)git_options_argument_optional}` vary without any reasons in this file. How about reformat them (quoting them should be better)?